README.md

```diff
@@ -1,3 +1,13 @@
+## Lab 8 Completion README
```

Comment:
Make your README more robust, i.e. instruct a user how to interact with your API using http commands.